static/openrpc.json

```diff
@@ -1139,7 +1139,9 @@
                 "type": "object",
                 "required": [
                   "type",
-                  "key"
+                  "key",
+                  "before",
```

Comment:
These are optional now right?

Reply:
No, see link in description: they're always present, sometimes null